documentation/source/quickstart.rst

```diff
@@ -4,291 +4,188 @@
-or a resolved integer value can be accessed using the :attr:`~cocotb.binary.BinaryValue.integer` attribute.
+* the default simulator to use (:make:var:`SIM`),
+* the default language of the toplevel module or entity (:make:var:`TOPLEVEL_LANG`, ``verilog`` in our case),
+* the design source files (:make:var:`VERILOG_SOURCES` and :make:var:`VHDL_SOURCES`),
+* the toplevel module or entity to instantiate (:envvar:`TOPLEVEL`,  ``my_design`` in our case),
+* and a Python module that contains our cocotb tests (:envvar:`MODULE`, ``test_my_design`` in our case).
 
-.. code-block:: pycon
+.. code-block:: makefile
 
-    >>> # Read a value back from the DUT
-    >>> count = dut.counter.value
-    >>> print(count.binstr)
-    1X1010
-    >>> # Resolve the value to an integer (X or Z treated as 0)
-    >>> print(count.integer)
-    42
-    >>> # Show number of bits in a value
-    >>> print(count.n_bits)
-    6
+    # This is file Makefile
 
-We can also cast the signal handle directly to an integer:
+    # defaults
+    SIM ?= icarus
+    TOPLEVEL_LANG ?= verilog
 
-.. code-block:: pycon
+    # assuming there are two implementations of the same design
+    # in the directory where this Makefile is located
+    VERILOG_SOURCES += $(PWD)/my_design.sv
+    VHDL_SOURCES += $(PWD)/my_design.vhdl
 
-    >>> print(int(dut.counter))
-    42
+    # TOPLEVEL is the name of the toplevel module in your Verilog or VHDL file
+    TOPLEVEL = my_design
 
+    # MODULE is the basename of the Python test file
+    MODULE = test_my_design
 
-Parallel and sequential execution
----------------------------------
+    # include cocotb's make rules to take care of the simulator setup
+    include $(shell cocotb-config --makefiles)/Makefile.sim
 
-An :keyword:`await` will run an :keyword:`async` coroutine and wait for it to complete.
-The called coroutine "blocks" the execution of the current coroutine.
-Wrapping the call in :func:`~cocotb.fork` runs the coroutine concurrently, allowing the current coroutine to continue executing.
-At any time you can :keyword:`await` the result of the forked coroutine, which will block until the forked coroutine finishes.
 
-The following example shows these in action:
+When you now type
 
-.. code-block:: python3
+.. code-block:: bash
 
-    # A coroutine
-    async def reset_dut(reset_n, duration_ns):
-        reset_n <= 0
-        await Timer(duration_ns, units='ns')
-        reset_n <= 1
-        reset_n._log.debug("Reset complete")
+   make
 
-    @cocotb.test()
-    async def parallel_example(dut):
-        reset_n = dut.reset
+Icarus Verilog will be used to simulate the Verilog implementation of the DUT because
+we defined these as the default values.
 
-        # Execution will block until reset_dut has completed
-        await reset_dut(reset_n, 500)
-        dut._log.debug("After reset")
+If you want to simulate the VHDL implementation of the DUT with GHDL instead,
```

Comment:
To make it more realistic, let's keep it to a single language. You can have this section then to show the use of a different simulator, e.g. vcs.

```suggestion
If you want to simulate the DUT with VCS instead,
```